static/app/components/globalFeedbackForm.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  createContext,
+  type ReactNode,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+} from 'react';
+import type {FeedbackDialog} from '@sentry/types';
+
+import {
+  type FeedbackIntegration,
+  useFeedback,
+  type UseFeedbackOptions,
+} from 'sentry/components/feedback/widget/useFeedback';
+
+interface GlobalFeedbackFormProps {
+  children: ReactNode;
+}
+
+interface GlobalFeedbackFormContextType {
+  feedback: FeedbackIntegration | undefined;
+  openForm: (options?: UseFeedbackOptions) => Promise<void>;
+}
+
+const GlobalFeedbackFormContext = createContext<GlobalFeedbackFormContextType>({
```

Comment:
+1 to simpler names. 

none of these are exported, so what if the names were `Props` `ContextType` and `const context = createContext(...);`